datalab/datalab.sh

```diff
@@ -38,6 +38,9 @@ if [[ "${ROLE}" == 'Master' ]]; then
     echo 'spark.sql.warehouse.dir=/root/spark-warehouse' >> "${SPARK_CONF}"
   fi
 
+  DATALAB_DIR="${HOME}/datalab"
```

Comment:
We don't preserve disks between clusters, so those notebooks would need to be manually up/downloaded.

I suppose this FR makes sense in any case, if anyone reloads the docker container, but the invocation to reload is arcane and undocumented.

I'll merge this as is and we can think about notebook staging later.